nuxeo-drive-core/src/test/java/org/nuxeo/drive/service/adapter/TestFileSystemItemAdapterService.java

```diff
@@ -638,14 +647,6 @@ public void testContribOverride() throws Exception {
         assertTrue(activeFactories.contains("dummyFacetFactory"));
         assertTrue(activeFactories.contains("dummyVirtualFolderItemFactory"));
         assertTrue(activeFactories.contains("nullMergeTestFactory"));
-
-        Framework.getRuntime().standby(Instant.now());
-        try {
-            harness.undeployContrib("org.nuxeo.drive.core.test",
```

Comment:
all deployments achieved in a test method are undeployed at tear down by the runtime feature. my understanding was that the undeploy was wrong because it was not inside a finally block.